jekyll/_components/tables.md

```diff
@@ -2,17 +2,142 @@
 layout: page
 title: Tables
 order: 9
-draft: true
 ---
 
-Tabular data appears in a variety of places, the Network panel is the most obvoious example. The console is also able to output a visual table using `console.table()` e.g. `console.table(["apples", "oranges", "bananas"]);`
-
-![console table example](../images/components/tables/console-table-example.png "Console table example")
+Tabular data appears in a variety of places, the Network panel is the most obvious example. The console is also able to output a visual table using `console.table()` e.g. `console.table(["apples", "oranges", "bananas"]);`
 
 ## Usage
 
+<div class="grid-2" markdown="1">
+![Usage Example of a Checkbox as used in Firefox Options](../images/components/tables/table-grid-light.png)
+
+<div markdown="1">
+Tables are used when data needs to be presented in a series of rows which may also posses any amount of columns. For large quantities of data it is recommended to include a button column header allowing users to sort the data by that specific column, as seen in the Network panel
+</div>
```

Comment:
I don't think I see the button column header in the screenshot... Am I missing it? Should we change the screenshot to one that includes it?

Reply:
In the image the "Status" and "Method" headers are the buttons, one could be highlighted for quick recognition however.